datasets/un_ga/README.md

```diff
@@ -4,12 +4,51 @@ annotations_creators:
 language_creators:
 - found
 languages:
-- ar
-- en
-- es
-- fr
-- ru
-- zh
+- ar-to-en
+  - ar
+  - en
```

Comment:
Can you remove the `-` in front of config names ? And add `:` after each config name ?
For example
```
  ar-to-en:
  - ar
  - en
```